Untitled.py

```diff
@@ -0,0 +1,53 @@
+
+
+
+def sheet_data(sheet):
+
+    out = list()
+
+    sheetIndex = 0
+    for X in range(1, s.nrows):
+        learner = s.row_values(X)
+        print(learner)
+        average=sum(learner)
+        out.append([learner[1], average])
+
+    return out
+
+def sum(learner):
+    average = 0
+    for score in range(2, len(learner)) :
+        average = average+learner[score] // 2
+        if debug>0:
+            print(learner[score], average)
+    return average
+
+import xlrd
+
+a = xlrd.open_workbook("БИОЛОГИ_ таблица успеваемости осень 2018.xlsx")
+debug = 1
+
+for s in a.sheets():
+    if (s.name == 'Python') == True:
+        out = sheet_data(s)
+        if debug == 1:
+            print(out)
+
+
+class PRESENT_RESULT  :
```

Comment:
no need in classes in this code

Reply:
Virtually any code can be rewritten with or without classes. Please explain, why do you think that classes are not necessary here?

Reply:
Just because any code can be written without classes

Reply:
So, you think that all Python code should be rewritten without classes?

Reply:
Not all obviously, but that one can work without it